src/test/java/com/hivemq/cli/rest/hivemq/TestResponseBodies.java

```diff
@@ -55,22 +59,28 @@
                     "    \"next\": \"/api/v1/hivemq/clients?cursor=bmV4dHJlc3VsdGFzZGprYXNkamFzbGRqYXM_\"\n" +
                     "  }\n" +
                     "}";
+            //@formatter:on
 
-    public static final String CLIENT_IDS_SINGLE_RESULT =
+    public static final @NotNull String CLIENT_IDS_SINGLE_RESULT =
+            //@formatter:off
             "{\n" +
                     "  \"items\": [\n" +
                     "    {\n" +
-                    "      \"id\": \"client-ݰ\"\n" +
+                    "      \"id\": \"client-π\"\n" +
```

Comment:
?

Reply:
I had absolutely no idea why this char was used in the first place but changed it to a utf letter which is not read right-to-left so intellij does not complain.